conda/plan.py

```diff
@@ -439,11 +439,11 @@ def add_defaults_to_specs(r, linked, specs, update=False, prefix=None):
 
 
 def install_actions(prefix, index, specs, force=False, only_names=None, always_copy=False,
```

Comment:
As far as I can tell nothing calls this function, so it can probably be ripped out too.